src/evict/evict_page.c

```diff
@@ -555,10 +575,6 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
     if (F_ISSET(session->dhandle, WT_DHANDLE_DEAD))
         return (0);
 
-    /* Temp fix: do not evict a page with active las */
```

Comment:
@keithbostic , after my changes that will let a page with history be evicted again, {{test_las06.py}} is failing. I am debugging the issue and will let you know if I make progress. The change is mostly to manage the page splits by aggregating the lookaside information from multi to new refs that get created.

I will let you know if I make progress in understanding the failure, otherwise, can you look at this change and figure out what could be the reason.

Reply:
@tetsuo-cpp : We talked about this here.